src/main/java/slimeknights/mantle/command/BookTestCommand.java

```diff
@@ -1,38 +1,30 @@
 package slimeknights.mantle.command;
 
 import com.mojang.brigadier.builder.LiteralArgumentBuilder;
-import com.mojang.brigadier.context.CommandContext;
-import com.mojang.brigadier.exceptions.CommandSyntaxException;
 import net.minecraft.commands.CommandSourceStack;
-import net.minecraft.commands.Commands;
-import net.minecraft.commands.arguments.ResourceLocationArgument;
 import net.minecraft.resources.ResourceLocation;
-import net.minecraft.server.level.ServerPlayer;
-import slimeknights.mantle.Mantle;
-import slimeknights.mantle.network.MantleNetwork;
-import slimeknights.mantle.network.packet.OpenNamedBookPacket;
 
 import java.util.HashSet;
 import java.util.Set;
 
-/** Command that opens a test book */
+/**
+ * Command that opens a test book
+ * @deprecated Command is now client-side and lives in {@link slimeknights.mantle.command.client.BookCommand}
+ */
```

Comment:
Should use the `@Deprecated` annotation in addition to the Javadocs or the Java build server yells at us